src/Resources/contao/dca/tl_page.php

```diff
@@ -520,6 +520,19 @@
 			'inputType'               => 'select',
 			'options'                 => array(0, 5, 15, 30, 60, 300, 900, 1800, 3600, 10800, 21600, 43200, 86400, 259200, 604800, 2592000),
 			'reference'               => &$GLOBALS['TL_LANG']['CACHE'],
+			'eval'                    => array('tl_class'=>'w50'),
+			'sql'                     => "int(10) unsigned NOT NULL default '0'"
+		),
+		'clientCache' => array
```

Comment:
I do understand that you want to set `setMaxAge()` and `setSharedMaxAge()` with different values, but what is the point?